modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring a namespace-scoped Argo Rollouts installation
+
+To install a namespace-scoped instance of Argo Rollouts, complete the following steps:
+
+.Prerequisites
+
+* You have logged in to the {product-title} cluster as an administrator.
+* You have installed {gitops-title} on your {product-title} cluster.
+
+.Procedure 
+
+. In the *Administrator* perspective of the web console, go to *Administration* -> *CustomResourceDefinitions*.
+. Search for `Subscription` and click the *Subscription* CRD.
+. Click the *Instances* tab and then click the *openshift-gitops-operator* subscription.
+. Click the *YAML* tab and edit the YAML file. 
++
+.. Specify the `NAMESPACE_SCOPED_ARGO_ROLLOUTS` environment variable, with the value set to `true` in the `.spec.config.env` property.
++
+.Example of configuring the namespace-scoped Argo Rollouts installation
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+spec:
+  # (...)
+  config:
+    env:
+      - name: NAMESPACE_SCOPED_ARGO_ROLLOUTS
+        value: 'true' #<1>
+----
+<1> Set value to `'true'` to enable namespace-scoped installation. If the value is set to `'false'` or not specified, the installation defaults to cluster-scoped mode.
```

Comment:
```suggestion
<1> The value set to `true` enables namespace-scoped installation. If the value is set to `'false'` or not specified the installation defaults to cluster-scoped mode.
```

About the quotation marks around the value true and false, please check with the SME and clarify whether to include the quotation marks or not.

Reply:
Thanks for the detailed explanation, @jgwest. Fixed.